samples/Server/src/routes/addUser.ts

```diff
@@ -12,6 +12,17 @@ interface AddUserParam {
   DisplayName: string;
 }
 
+/**
+ * route: /addUser/[threadId]
+ *
+ * purpose: Add the user to the chat thread with given threadId.
+ *
+ * @param threadId: id of the thread to which user needs to be added
+ * @param id: id of the user
```

Comment:
Would be nice to document the format of the user ID. Is this ACS user ID as string?